Libraries/LibGfx/ImmutableBitmap.cpp

```diff
@@ -60,6 +60,7 @@ Color ImmutableBitmap::get_pixel(int x, int y) const
     return m_impl->source.get<NonnullRefPtr<Gfx::Bitmap>>()->get_pixel(x, y);
 }
 
+// Fixme: duplicates code from PaintingSurface.cpp
```

Comment:
Why not introduce a shared private header file for this to centralize implementations?

Reply:
Because I don't really understand a concept of "shared private header" :(

Reply:
You could introduce something like `Libraries/LibGfx/SkiaUtils.h`, implement the shared functions there, and include that header in both `.cpp` files (not the `.h`s!)

Reply:
Sounds like a good idea. I hadn’t considered introducing a new header; I was more focused on trying to use one of the existing ones.

Reply:
I will do that and return

Reply:
As an example of where we do this elsewhere, see [Libraries/LibUnicode/ICU.h](https://github.com/LadybirdBrowser/ladybird/tree/master/Libraries/LibUnicode/ICU.h). That header includes ICU headers, but is itself only included by .cpp files in LibUnicode, so the ICU headers don't "leak".

Reply:
Now it fails to build some Swift stuff on Mac. Again :(